lib/execution_engine2/execution_engine2Server.py

```diff
@@ -12,8 +12,13 @@
 from wsgiref.simple_server import make_server
 
 import requests as _requests
-from jsonrpcbase import JSONRPCService, InvalidParamsError, KeywordError, \
```

Comment:
I didn't read through changes in this file. I assume it's an auto linter does all the changes. It shouldn't probably turn off on sdk auto-generated files.

Reply:
Right now it just skips the impl file

https://github.com/kbase/execution_engine2/blob/develop/.pre-commit-config.yaml